.github/workflows/validate-pr.yml

```diff
@@ -44,3 +44,16 @@ jobs:
 
           echo "This PR will be included in the release notes with the following note:"
           echo "$desc"
+
+  # If PR title starts with [{branch} backport] - make sure that it's opened against the {branch} branch
+  check-pr-title:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check backport branch
+        if: startsWith(github.event.pull_request.title, '[') && contains(github.event.pull_request.title, ' backport]')
```

Comment:
Maybe we don't even need to check for ` backport` because there's also PRs that (at times) may go directly to a release branch  (thinking of, e.g., Go updates where "master" already moved to a new version, so theres' nothing to cherry-pick).


In that case, we would have to check for the first word (non-space) between `[ ]`.

Reply:
Will do!

Reply:
Done.